### ISSUE TITLE
[Angular 10] Keyboard navigation broken

### ISSUE THREAD

**Issue by eXalleR-** [h2qutc/angular-material-components#374]
Just open this demo, then open the calendar and try to change the date by keyboard arrow keys.
https://stackblitz.com/edit/demo-ngx-mat-datetime-picker-angular10?file=src%2Fapp%2Fapp.component.ts

BTW it works on 9x and 17x
9x: https://stackblitz.com/edit/demo-ngx-mat-datetime-picker-angular9?file=src%2Fapp%2Fapp.component.ts (the only focus CSS is missing, but keyboard  navigation works)
17x: https://stackblitz.com/edit/demo-ngx-mat-datetime-picker?file=src%2Fapp%2Fapp.component.ts (the only focus CSS is missing, but keyboard navigation works)
